lisa/features/infiniband.py

```diff
@@ -269,10 +271,14 @@ def setup_rdma(self) -> None:  # noqa: C901
                     )
                     node.os.install_packages("kernel-devel")
         elif isinstance(node.os, Ubuntu) and node.os.information.version >= "18.4.0":
-            if node.os.information.version >= "22.10.0":
-                ubuntu_required_packages.append("lib32gcc-9-dev python3-dev")
-            else:
-                ubuntu_required_packages.append("lib32gcc-8-dev python-dev")
+            for package in [
```

Comment:
Why change to this way? It may install more versions than needed. And the progress is slower than install them together.

Reply:
it will fail to install package, if any one of package list can't not be found in repo.